pkgs/top-level/all-packages.nix

```diff
@@ -14143,7 +14143,7 @@ with pkgs;
 
   lightum = callPackage ../os-specific/linux/lightum { };
 
-  ebtables = callPackage ../os-specific/linux/ebtables { };
+  ebtables = callPackage ../os-specific/linux/ebtables { enableStatic = stdenv.hostPlatform.isMusl; };
```

Comment:
I would move the default inside the package itself

```
{
  ...
  enableStatic = stdenv.hostPlatform.isMusl
}
```

Otherwise people might miss the override from `all-packages.nix`